wrecker.go

```diff
@@ -84,6 +104,20 @@ func (w *Wrecker) sendRequest(verb string, requestURL string, headers map[string
 	if err != nil {
 		return nil, err
 	}
-	err = json.Unmarshal(body, &response)
+
+	err = json.Unmarshal(body, r.Response)
 	return resp, err
 }
+
+// prepareRequestBody() function was originally included in the
```

Comment:
I would actually say we can remove this comment now as this function has deviated so much.